### PR TITLE
[CI:DOCS] Document --userns=auto behaviour for rootless users

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by giuseppe — APPROVED** [review]
LGTM

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL

File: docs/source/markdown/options/userns.container.md [hunk 1]
```diff
@@ -30,9 +30,11 @@ Valid _mode_ values are:
 
 **auto**[:_OPTIONS,..._]: automatically create a unique user namespace.
 
-The `--userns=auto` flag requires that the user name __containers__ be specified in the /etc/subuid and /etc/subgid files, with an unused range of subordinate user IDs that Podman containers are allowed to allocate. See subuid(5).
+* `rootful mode`: The `--userns=auto` flag requires that the user name __containers__ be specified in the /etc/subuid and /etc/subgid files, with an unused range of subordinate user IDs that Podman containers are allowed to allocate.
 
-Example: `containers:2147483647:2147483648`.
+  	   Example: `containers:2147483647:2147483648`.
+
+* `rootless mode`: The users range from the /etc/subuid and /etc/subgid files will be used. Note if you run a single container without using --userns=auto, it will use the entire range of UIDs and not allow further subdeviding. See subuid(5).
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/options/userns.container.md:37]:
```suggestion
* `rootless mode`: The users range from the /etc/subuid and /etc/subgid files will be used. Note running a single container without using --userns=auto will use the entire range of UIDs and not allow further subdividing. See subuid(5).
```

**Comment by TomSweeneyRedHat** [comment]
Couple tweaks, otherwise LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**